python/pfs/drp/stella/calibrateWavelengthsTask.py

```diff
@@ -15,26 +15,33 @@
 
 class LineData(SimpleNamespace):
     """Data for a single reference line
-
     Parameters
     ----------
     fiberId : `int`
         Fiber identifier.
-    pixels : `float`
-        Pixel position of line in spectrum.
-    measuredWavelength : `float`
-        Measured wavelength of line in spectrum (nm).
-    actualWavelength : `float`
-        Actual wavelength of line (nm).
+    reflines : 'float'
+        reference line wavelength (nm).         
+    nominalPixelPos : `float`
+        Pixel position calculated with detectormap.
+    fitPixelPos : `float`
+        Pixel position of reference line determined by the fit
+    fitWavelength : 'float'
+        wavelength determined by the fit position.
+    fitPixelPosErr : `float`
+        Error of fit fitPixelPos 
+    fitWavelengthErr : 
+        Error of fit Wavelength
+    wavelengthCorr : 'float'
+        Chevichev fit result 
+    status : int
```

Comment:
Put type in backticks. Is it actually an int, or is the type `pfs.drp.stella.ReferenceLine.Status` (which can work like an int, but is actually a C++ enum)?